src/import-utils.ts

```diff
@@ -94,6 +117,7 @@ export async function inlineHtmlImport(
   }
 
   const importDoc = parse5.parseFragment(importSource);
+  debaseDocument(resolvedImportUrl, importDoc);
```

Comment:
Why is debaseDocument a pass on the document, rather than just a part of `rewriteImportedUrls`?